src/main/clm_instMod.F90

```diff
@@ -268,6 +271,14 @@ subroutine clm_instInit(bounds)
          urbanparams_inst%thick_wall(begl:endl), &
          urbanparams_inst%thick_roof(begl:endl))
 
+    ! Set hillslope column bedrock values
+    if (use_hillslope) then
+       call SetHillslopeSoilThickness(bounds,fsurdat, &
+            soil_depth_lowland_in=8.5_r8,&
```

Comment:
What's the meaning of the hardcoded constants here (8.5 and 2.0)? Does it improve readability to make them a parameter?